CHANGES.md

```diff
@@ -1,3 +1,18 @@
+## edge-19.4.1
+
+* CLI
+  * Introduced an `upgrade` command! This allows an existing Linkerd Control Plane to be reinstalled or reconfigured.
```

Comment:
Everything looks good to me - one styling nit, remove the period from the end of this bullet point

Reply:
Haha should have reloaded the page, @siggy beat me to it